k8s/pkg/apis/longhorn/v1beta2/volume.go

```diff
@@ -110,6 +110,15 @@ const (
 	SnapshotDataIntegrityFastCheck = SnapshotDataIntegrity("fast-check")
 )
 
+// +kubebuilder:validation:Enum=ignored;enabled;disabled
+type RestoreVolumeRecurringJobOpt string
```

Comment:
Rename to `RestoreVolumeRecurringJobType`

Reply:
Modified and all related codes. Thanks